src/Atmos/Model/turbulence.jl

```diff
@@ -95,18 +95,18 @@ end
 
 vars_gradient(::ConstantViscosityWithDivergence,FT) = @vars()
 vars_diffusive(::ConstantViscosityWithDivergence, FT) =
-  @vars(S::SHermitianCompact{3,FT,6})
+  @vars(∇u::SMatrix{3,3,FT,9}) #@vars(S::SHermitianCompact{3,FT,6})
 
 function diffusive!(::ConstantViscosityWithDivergence, ::Orientation,
     diffusive::Vars, ∇transform::Grad, state::Vars, aux::Vars, t::Real)
-
-  diffusive.turbulence.S = symmetrize(∇transform.u)
+    diffusive.turbulence.∇u = ∇transform.u
+  #diffusive.turbulence.S = symmetrize(∇transform.u)
```

Comment:
Please remove commented lines of code once you're certain the change is correct, here and in numerous places throughout.

Reply:
This file is reset to master.